ui.frontend/src/view/FormTabs.js

```diff
@@ -237,11 +241,16 @@ class FormTabs extends FormPanel {
      * @param {string} tabId - The ID of the tab to navigate to.
      */
     navigateAndFocusTab(tabId) {
-        this.navigate(tabId);
-        this.focusWithoutScroll(this.#getTabNavElementById(tabId));
+        this.setActive();
+        // if current active tab and the new tabId is not same, only then set the new tab as active
+        if (this.#_active !== tabId) {
```

Comment:
A tab needs to be only set as active, if it not already active, this was auto-triggering blur on other fields inside an active tab (due to multiple focus event being triggered on tab)